src/Views/Nav.elm

```diff
@@ -30,7 +30,7 @@ view _ =
             , span [ class "nav-item" ]
                 [ a [ class "btn ", href "https://google.com", target "_blank", rel "noopener", onClick (Exit True) ]
                     [ getIcon "exit-door" (Just "nav-item-text nav-icon")
-                    , span [ class " nav-item-text" ] [ text "Exit Site" ]
+                    , span [ class " nav-item-text text-selector" ] [ text "Exit Site" ]
```

Comment:
Can we give this class a name that will help identify what kind of text it's selecting?

Reply:
sure, nav-text-selector maybe?